Docs/reference/content/reference/driver/crud/linq.md

```diff
@@ -69,6 +69,15 @@ var query = collection.AsQueryable()
 ]
 ```
 
+Also, there is a list of aggregate options that can be applied for the whole pipeline via [`AggregationOptions`]({{< apiref "T_MongoDB_Driver_AggregateOptions" >}}). Below is an example of [`Collation`]({{< docsref "reference/collation/" >}}) configuring:
```

Comment:
Maybe fix grammar slightly:

    Below is an example of configuring a [`Collation`]({{< docsref "reference/collation/" >}}):

Reply:
done